src/plugins/intel_npu/src/backend/src/zero_backend.cpp

```diff
@@ -20,6 +20,14 @@ ZeroEngineBackend::ZeroEngineBackend(const Config& config) {
     _devices.emplace(std::make_pair(device->getName(), device));
 }
 
+uint32_t ZeroEngineBackend::getDriverVersion() const {
+    return _driver_properties.driverVersion;
```

Comment:
Where do we populate this? Should it be stored in ZeroInitStructsHolder as well and exposed through _instance->GetDriverVersion() ?

Reply:
good catch, thank you! done